doc/fastly-configuration.rst

```diff
@@ -0,0 +1,43 @@
+.. _fastly configuration:
+
+Fastly Configuration
+-------------------
+
+Caching is enabled by default. All requests that are marked as cacheable will be cached.
+
+Tagging
+~~~~~~~
+
+Fastly supports :term:`tagging` out of the box.
+Configure the tag header to ``Surrogate-Keys``. (``fos_http_cache.tags.response_header`` if you use FOSHttpCacheBundle)
+
+Purge
+~~~~~
+
+Fastly supports two types of deletion of cached items using tags:
+
+1. Soft Purge: This is the default behavior. It will mark the cached item as stale. Stale items *may* still be served (e.g. with the ``stale-while-revalidate`` or ``stale-on-error`` cache control headers. Once the TTL expires, the stale item will be evicted from the cache and the next request must be fetched from the origin.
```

Comment:
```suggestion
1. Soft Purge: This is the default behavior. It will mark the cached item as stale. Stale items *may* still be served (e.g. with the ``stale-while-revalidate`` or ``stale-on-error`` cache control headers). Once the TTL expires, the stale item will be evicted from the cache and the next request must be fetched from the origin.
```